src/index.html

```diff
@@ -7,7 +7,54 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <stars class="stars stars--0">
+      <star class="stars__star"></star>
+      <star class="stars__star"></star>
+      <star class="stars__star"></star>
+      <star class="stars__star"></star>
+      <star class="stars__star"></star>
+    </stars>
```

Comment:
the `star` and `stars` tags do not exist, they must be existing tags for semantics